src/main/java/org/example/controller/DeveloperController.java

```diff
@@ -62,11 +66,34 @@ public Response getJsonType(Arguments arguments) {
         return response;
     }
 
+    public Response create(Arguments arguments) {
+        Response response = new Response();
+
+        Developer developer = argumentToDeveloperConverter.convert(arguments);
+        String jsonString = new Gson().toJson(developer);
```

Comment:
Why you convert developer into JSON and then convert JSON into developer? (lines 73 / 78)